projects/multibit/index.js

```diff
@@ -0,0 +1,47 @@
+const abi = require("./abi.json");
+const { getConfig } = require('../helper/cache')
+const ADDRESSES = require('../helper/coreAssets.json')
+const { sumTokensExport } = require('../helper/chain/brc20')
+
+const BRIDGE_TOKENS = "https://api.multibit.exchange/support/token";
+
+const config = {
+  // https://app.multibit.exchange/staking
+  ethereum: { stakingPool: "0x2EDfFbc62C3dfFD2a8FbAE3cd83A986B5bbB5495", tokens: [ADDRESSES.null, ADDRESSES.ethereum.USDT] },
+  bsc: {},
+  polygon: {},
+}
+module.exports = {
+  methodology: `Tokens bridged via MultiBit are counted as TVL`,
+  bitcoin: {
+    tvl: sumTokensExport({
+      owners: [
+        'bc1p6r6hx759e3ulvggvd9x3df0rqh27jz59nvfjd2fzmh3wqyt6walq82u38z', // hot wallet
+        'bc1pyyms2ssr0hagy5j50r5n689e6ye0626v3c98j5fw0jk6tz3vrgts7nt56g',  // cold wallet
+        'bc1qmcrpqanjnrw58y0fvq08fqchgxv5aylctew7vxlkalfns3rpedxsx4hxpu',  // cold wallet
+      ],
+      blacklistedTokens: ['MUBI', 'BSSB']
```

Comment:
Hi @g1nt0ki :
These assets are users’ cross-chain assets and do not belong to the project itself.
In particular $BSSB is an external asset and should not be in the blacklist.

Reply:
hi @pipi-multibit I blacklisted it because it is linked with the project, it is standard practice when it comes to counting tvl: 

https://github.com/DefiLlama/DefiLlama-Adapters/pull/3575/commits/14beefe0dfce99cbcedff51f8fef3ed1249ac7c6#diff-def88ffb911cabedad4293f2b43b47f3acb6b72ac3bf62636bbf80ad5bc8688bR82

https://github.com/DefiLlama/DefiLlama-Adapters/pull/6273/commits/38c49f6c09e614536d0d87bab05253dc3bc13ef2